scripts/create_resources.sh

```diff
@@ -24,12 +24,21 @@ gcloud iam service-accounts create $SERVER_NAME \
 gcloud projects add-iam-policy-binding $PROJECT \
        --member="serviceAccount:$SERVER_ACCOUNT" \
        --role='roles/lifesciences.workflowsRunner' > /dev/null
```

Comment:
Should we remove the lifesciences one here?

Reply:
Yes, I think this makes sense